certbot-apache/certbot_apache/display_ops.py

```diff
@@ -13,10 +13,42 @@
 logger = logging.getLogger(__name__)
 
 
+def select_vhost_multiple(vhosts):
+    """Select multiple Vhosts to install the certificate for
+
+    :param vhosts: Available Apache VirtualHosts
+    :type vhosts: :class:`list` of type `~obj.Vhost`
+
+    :returns: List of VirtualHosts
+    :rtype: :class:`list`of type `~obj.Vhost`
+    """
+    if not vhosts:
+        return list()
+    tags_list = [vhost.display_repr() for vhost in vhosts]
+    while True:
```

Comment:
I presumed there being an undefined reason behind this, as `select_vhost()` function has the same behavior. It's been around since original implementation by James, introduced in 99fcb4f23 and now looking at the original commit, it looks like it was in place for a help selection that has since been removed. I cleaned this up from the `select_vhost()` too.